communities/microgalaxy/lab/sections/2_microbial_isolates.yml

```diff
@@ -301,6 +327,18 @@ tabs:
       - id: genomics
```

Comment:
Can you add from https://github.com/galaxyproject/galaxy_codex/blob/main/communities/microgalaxy/resources/curated_workflows.tsv, the worklows on line 7, 14, 15, 18, 32?

Reply:
15 was already there, I added the rest